Sources/BraintreePayPal/BTPayPalVaultErrorHandlingEditRequest.swift

```diff
@@ -16,9 +18,17 @@ public struct BTPayPalVaultErrorHandlingEditRequest {
     ///   - riskCorrelationID: Required: Unique id for each transaction used in subsequent retry in case of failure
     ///   - merchantAccountID: Optional: ID of the merchant account; if one is not provided the default will be used
     /// - Warning: This feature is currently in beta and may change or be removed in future releases.
-    public init(editPayPalVaultID: String, riskCorrelationID: String, merchantAccountID: String? = nil) {
-        self.editPayPalVaultID = editPayPalVaultID
-        self.merchantAccountID = merchantAccountID
+    public init(editPayPalVaultID: String, riskCorrelationID: String) {
         self.riskCorrelationID = riskCorrelationID
+        super.init(editPayPalVaultID: editPayPalVaultID)
+    }
+
+    public func parameters() -> [String: Any] {
```

Comment:
I don't think we need this here since it exists in the super class unless we aren't supposed to pass `risk_correlation_id` for error handling

Reply:
Made corrections to omit this.